libbpf-rs/src/program.rs

```diff
@@ -344,4 +360,15 @@ impl Program {
             Ok(Link::new(ptr))
         }
     }
+
+    /// Attach this program to [sk_lookup](https://lwn.net/Articles/819618/)
+    pub fn attach_sk_lookup(&mut self, netns_fd: i32) -> Result<Link> {
```

Comment:
wait, I thought you are going to add "attach_netns" method... Why deviate from the libbpf naming? sk_lookup is just one subtype of programs that can be attached here. I think flow_dissector is the other one right now? This method should support both out of the box, no? So sk_lookup-specific name is suboptimal.

Reply:
Oops, sorry, I misunderstood your previous comment. The method is now called `attach_netns`. I agree this is better.

Reply:
Oh, the MR is already merged. I'll open a new one to rename the method.

Reply:
I'll fix it up locally, don't send another one